### DISCUSSION TITLE
How can I obtain metrics such as the "code acceptance rates" for GitHub Copilot?

### DISCUSSION THREAD

**Discussion by matsuo-keita** [community/community#109127] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Hi! Various large Japanese companies have published data on the acceptance rate of code when using GitHub Copilot. [This article](https://buildersbox.corp-sansan.com/entry/2023/09/01/110000) (sorry it's in Japanese) mentions that they requested this data from GitHub. Can any company receive such data if they request it? And if possible, through which form should we submit our request?

<img width="625" alt="image" src="https://github.com/community/community/assets/77378494/6d831a8d-23f8-46ba-9fcd-370f29e6e72d">

**Comment by davecheney** [comment]
Please have the org admin for your company contact your sales representative.

**Comment by davecheney** [comment, marked as answer]
Duplicate of https://github.com/orgs/community/discussions/54691
